client/src/components/Dashboard/DashboardStyledComponent.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components"
+
+const PageDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 800px;
+
+  @media only screen and (min-width: 800px) {
+    margin: auto;
+  }
+`
+
+const DashboardDiv = styled.div`
```

Comment:
we should try to make the names more descriptive.
```suggestion
const DashboardCard = styled.div`
```